docs/node_guide.md

```diff
@@ -0,0 +1,148 @@
+# Node Guide
+
+This is an updated guide on setting up a mainnet node. Note that this is a minimal guide and does not cover more advanced topics like [sentry node architecture](https://github.com/stakefish/cosmos-validator-design) and [double signing protection](https://github.com/tendermint/tmkms). It is strongly recommended that any parties considering validating do additional research.  If you have questions, please join the active conversation in the #validators thread of our [__Discord Channel__](https://discord.com/invite/kQzh3Uv).
```

Comment:
Can cut the part about "minimal guide". 

"This is an updated guide on setting up a mainnet full or light node."